controller/podcontroller.go

```diff
@@ -221,7 +221,12 @@ func (c *PodController) checkReadyStatus(oldpod, newpod *apiv1.Pod, cluster *crv
 							cluster.ObjectMeta.Namespace)
 					}
 
-					operator.UpdatePghaDefaultConfigInitFlag(c.PodClientset, false, clusterName, newpod.ObjectMeta.Namespace)
+					err = operator.UpdatePGHAConfigInitFlag(c.PodClientset, false, clusterName,
+						newpod.ObjectMeta.Namespace)
+					if err != nil {
+						log.Error(err)
+						return
+					}
```

Comment:
Suggest condensing this to `if err := ... ; err != nI { ... }`

Reply:
I tend to keep them separate (as seen here) when the expression for calling the function is long, e.g. do to a long function name, lots of params, etc., so this was actually a conscious decision here.  IMO it just feels cleaner/easier to read in certain situations.

Definitely some grey area with these things though and I'm not too passionate one way or another, i.e. if we want to establish that the initialization statement approach should _always_ be utilized.

Reply:
IMV even if it's long, it does control scope,and when it's too long, I have the `if` statement take up two lines :smile:

Reply:
Yeah, I'm not necessarily a proponent of _always_ (I don't think), as I'm sure there are times that it doesn't make sense. But, FWIW, when it's long I might do something like:

```
if err := someOperation(
  with,
  a,
  long,
  parameter,
  list,
); err != nil {
  // handle error
}
```

The benefit, IMO, is that:

1. it's easier to determine and document what's being passed to the function. For instance: what does `false` mean or set? Or what if `nil` or `""` is passed to a function? Sure, there are docs for that (right? :thinking: :wink:) but, bringing that context into the documentation/comment for such a call, helps for long term/overall maintenance, IMO.
2. It's also easier to modify the parameter list if it changes (for some reason) as well as identify what changed in a `diff`.

Granted, I'm not so opinionated in this case as to insist one way or the other. It's just my preference. :-)